src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +78,17 @@ public static Command duplicate(Command command) {
                 command.getRequirements().toArray(Subsystem[]::new)
         );
     }
+
+    public static Command getClimbCommand() {
+        return new SequentialCommandGroup(
+                new InstantCommand(
+                        () -> {
+                            CommandConstants.IS_CLIMBING = true;
+                            Logger.recordOutput("IsClimbing", true);
+                        }
+                ),
+                ClimberCommands.getSetTargetStateCommand(ClimberConstants.ClimberState.PREPARE_FOR_CLIMB),
+                runWhenContinueTriggerPressed(ClimberCommands.getSetTargetStateCommand(ClimberConstants.ClimberState.CLIMB))
```

Comment:
The logic here is flawed. This is why I want to see a full climb sequence- proof of concept.